bot.ts

```diff
@@ -7,19 +7,25 @@
 import * as djs from "discord.js"
 import * as djsvoice from "@discordjs/voice"
 
-const DBM: any = {};
-DBM.version = "2.2.0";
+interface DBM {
+	version: string;
+	DiscordJS: typeof djs;
+	requiredDjsVersion: string;
+}
 
-const DiscordJS = (DBM.DiscordJS = require("discord.js"));
+const DBM: DBM = {
+	version: (await import("./package.json")).default.version;
+	DiscordJS: (await import("discord.js"));
+	requiredDjsVersion: "14.11.0";
```

Comment:
In my mind separating these out of the initial object was a waste of time I know `requiredDjsVersion` wasn't on the object before but to be honest its not a bad property to have on it, if you want it separated still we can do that.